.github/workflows/test-library.yml

```diff
@@ -772,4 +772,38 @@ jobs:
         to the just published release back to GitHub
       run: >-
         git push --atomic origin '${{ needs.pre-setup.outputs.git-tag }}'
+
+  publish-github-release:
+    name: >-
+      Publish a GitHub Release for
+      ${{ needs.pre-setup.outputs.git-tag }}
+    needs:
+    - post-release-repo-update
+    - pre-setup  # transitive, for accessing settings
+    runs-on: Ubuntu-latest
+
+    permissions:
+      contents: write
+
+    steps:
+    - name: Download all the dists
+      uses: actions/download-artifact@v2
+      with:
+        name: python-package-distributions
+        path: dist/
+
+    - name: >-
+        Publish a GitHub Release for
+        ${{ needs.pre-setup.outputs.git-tag }}
+      uses: softprops/action-gh-release@v1
+      with:
+        # body/body_path:  # FIXME: Use once Towncrier is integrated.
+        discussion_category_name: Announcements
```

Comment:
@abhinavsingh this is why I asked you to create the discussions category. It'll auto-create announcement topics on release so that there's a centralized place for people to comment on a given version post-release.

Reply:
Awesome, I haven't checked on the release flows yet . Good to see.  Was thinking of release this week, but might only be able to make one over the weekend.  Will keep ya updated or anyways it will post to announcements :D

Reply:
@abhinavsingh if you want, we could plan to do this via a video call together. LMK if you'd want to do this and the TZ you're currently in.